server/Model/user.js

```diff
@@ -0,0 +1,52 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const userSchema = new Schema(
+  {
+    firstName: {
+      type: String,
+      required: true,
+    },
+    lastName: {
+      type: String,
+      required: true,
+    },
+    email: {
+      type: String,
+      required: true,
+    },
+    time_zone: {
+      type: String,
+    },
+    country_code: {
+      type: String,
+    },
+    expert_meeting: {
+      type: Number,
+      required: true,
+      default: 0,
+    },
+    learner_meeting: {
+      type: Number,
+      required: true,
+      default: 0,
+    },
+    challeneges_added: {
```

Comment:
This can be inferred from their records, keeping a value like this synced will be extra work i believe.
Same with expert meetings, learner meetings and challenges complete

Reply:
That way, every time we add a meeting or challenge, we have to go through all the records, filters, and count (e.g. challenges). This way is a little faster. While Adding a challenge, we can just increment the number in `user` Schema.